Automattic/jetpack
# [Plugin] Backup: Add No Backup Capabilities screen

#### Changes proposed in this Pull Request:

* Implements design for the screen presented when a user has no backup capabilities
* Removes some unused CSS
* Fixes an issue where AM / PM would wrap to a new line in latest backups

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Install the backup plugin on a site
* If the site has a backup plan, remove it via Store Admin ( You may have to wait for capabilities cache to expire )
* Ensure the No Backup Plan screen displays like the screenshot below
* Screen should be responsive (adjust width to ensure the layout has no issues)
